classes/class-wc-connect-taxjar-integration.php

```diff
@@ -913,6 +913,8 @@ public function append_base_address_to_customer_taxable_address( $address ) {
 	 * @return object
 	 */
 	public function maybe_override_taxjar_tax( $taxjar_resp_tax, $body ) {
+		if( ! isset($taxjar_resp_tax) ) return;
```

Comment:
WordPress Coding standards: Inline control structures are not allowed.
```suggestion
		if ( ! isset( $taxjar_resp_tax ) ) {
			return;
		}
```